scripts/stf/filters.json

```diff
@@ -0,0 +1,43 @@
+[
```

Comment:
@kbromer @mpusto @davidhabib Can you please review this list of keys that are to be filtered out from the raw export file?

This is copied over from the Google sheet, more or less, but I also added 'Scontrol' related keys, since including those will cause the import to fail (the Scontrols do not and can not exist in unmanaged deployments of the underlying packages)